src/app/content/components/Sidebar.tsx

```diff
@@ -204,5 +211,8 @@ export default connect(
   (state: AppState) => ({
     ...selectors.bookAndPage(state),
     isOpen: selectors.tocOpen(state),
+  }),
+  (dispatch: Dispatch) => ({
+    onNavigate: () => dispatch(resetToc()),
```

Comment:
It seems that the ToC collapses _before_ the page changes; is that desired? It seems a bit confusing if the user is on a slow internet connection.

Reply:
i think the ideal solution would be to add a loading state to the content side of the page. given that i think its better to close the toc immediately after clicking

Reply:
@snm8 Are there plans to design a `Loading...` UI when the user navigates between pages? I can create a story if not. It's not critical but it is one of the weird intermediate states that rex can be in.